packages/server/graphql/mutations/createTaskIntegration.ts

```diff
@@ -61,92 +67,91 @@ export default {
         {userId: viewerId}
       )
     }
-    const {repoOwner, repoName} = GitHubRepoId.split(nameWithOwner)
-    if (!repoOwner || !repoName) {
-      return standardError(new Error(`${nameWithOwner} is not a valid repository`), {
-        userId: viewerId
-      })
+
+    const integrationManager = TaskIntegrationManagerFactory.getManager(integrationProviderService)
+
+    if (!integrationManager) {
+      return standardError(new Error('Integration provider is not supported'))
     }
 
-    const {content: rawContentStr, meetingId} = task
+    const authDataLoader = dataLoader.get(integrationManager.authLoaderKey)
```

Comment:
@igorlesnenko 
> as I see for Jira server we load auth like
> ```
> return dataLoader
>  .get('teamMemberIntegrationAuths')
>  .load({service: 'jiraServer', teamId, userId})
> ```
> While for jira there is no server param
> ```
> parent.get('freshAtlassianAuth').load({teamId, userId})
> ```
> Maybe I should revert here to not return authLoaderKey but the whole auth loader?
> https://github.com/ParabolInc/parabol/pull/5938#discussion_r793422311

Very good observation, I think what could be a good solution is moving the `dataLoader.get` part into the `TaskIntegrationManagerFactory` so we get

```ts
const viewerIntegrationManager = TaskIntegrationManagerFactory.initManager(dataLoader, integrationProviderService, {teamId, userId})
const assigneeIntegrationManager = !viewerIntegrationManager && TaskIntegrationManagerFactory.initManager(dataLoader, integrationProviderService, {teamId, userId})
```

Reply:
As discussed, in place this would look like
```ts
    const [viewerIntegrationManager, assigneeIntegrationManager, team, teamMembers] = await Promise.all([
      TaskIntegrationManagerFactory.initManager(dataLoader, integrationProviderService, {teamId, userId})
      userId ? TaskIntegrationManagerFactory.initManager(dataLoader, integrationProviderService, {teamId, userId}) : null,
      dataLoader.get('teams').load(teamId),
      dataLoader.get('teamMembersByTeamId').load(teamId)
    ])
    const integrationManager = viewerIntegrationManager ?? assigneeIntegrationManager
    const accessUserId = viewerIntegrationManager ? viewerId : assigneeIntegrationManager ? userId : null

    if (!integrationManager) {
      return standardError(new Error('No auth exists for a given task!'), {userId: viewerId})
    }
```